docs/wiki/[Variables]-configure_management_resources.md

```diff
@@ -1 +1,355 @@
+
+#### `settings.security_center.config`
+
+The `config` (`object`) input allows you to set the following configuration items:
+
+##### `settings.security_center.email_security_contact`
+
+Specifies the email address to be used as the security contact in Microsoft Defender for Cloud.
+
+##### `settings.security_center.enable_defender_for_acr`
+
+Enables the the Standard pricing tier for `ContainerRegistry`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_app_services`
+
+Enables the the Standard pricing tier for `AppServices`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_arm`
+
+Enables the the Standard pricing tier for `Arm`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_dns`
+
+Enables the the Standard pricing tier for `Dns`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_key_vault`
+
+Enables the the Standard pricing tier for `KeyVaults`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_kubernetes`
+
+Enables Microsoft Defender for Cloud for all in-scope Azure Kubernetes Service clusters.
+This is deployed to all in-scope clusters using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_oss_databases`
+
+Enables the the Standard pricing tier for `OpenSourceRelationalDatabases`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_servers`
+
+Enables the the Standard pricing tier for `VirtualMachines`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_sql_server_vms`
+
+Enables the the Standard pricing tier for `SqlServerVirtualMachines`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_sql_servers`
+
+Enables the the Standard pricing tier for `SqlServers` (Azure SQL instances).
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+##### `settings.security_center.enable_defender_for_storage`
+
+Enables the the Standard pricing tier for `StorageAccounts`.
+This is deployed to all in-scope subscriptions using the `DeployIfNotExists` policy effect.
+
+### Location
+
+This allows the default location for all management resource to be set.
+Will override the `default_location` property set in the module.
```

Comment:
May also be overridden when set at more specific scope using the advanced config options, but not sure we want to document that here, and probably more relevant to call out on the `configure_connectivity_resources` page:

- #187

Reply:
WIll add clarification